src/olympia/accounts/tests/test_helpers.py

```diff
@@ -1,23 +1,31 @@
 from django.test import RequestFactory
 
-from unittest import mock
-
 from olympia.accounts.templatetags import jinja_helpers
 
 
-@mock.patch(
-    'olympia.accounts.templatetags.jinja_helpers.utils.default_fxa_login_url',
-    lambda c: 'http://auth.ca',
-)
 def test_login_link():
     request = RequestFactory().get('/en-US/firefox/addons')
-    assert jinja_helpers.login_link({'request': request}) == ('http://auth.ca')
+    assert jinja_helpers.login_link({'request': request}) == (
+        'http://testserver/api/v5/accounts/login/start/'
```

Comment:
Would it be worth putting `'http://testserver/api/v5/accounts/login/start/'` into a constant as it's used in each of the three assertions below? Also, how do we know that is the correct URL to expect? Can that be imported from elsewhere to be sure it's the correct value?

Reply:
If you don't hardcode the expected result, then you end up re-creating the logic from the function being tested (which is super short - it's just concatenating the result of `drf_url()` with `path_with_query()` encoded).

I could add a constant, but I'm not super convinced it's necessary since it's the same block of code and it's in a test.

Reply:
> If you don't hardcode the expected result, then you end up re-creating the logic from the function being tested (which is super short - it's just concatenating the result of `drf_url()` with `path_with_query()` encoded).
> 
> I could add a constant, but I'm not super convinced it's necessary since it's the same block of code and it's in a test.

Fair enough. We tend to use constants whenever possible in tests for frontend, but I know that's not the case for server, so this is consistent with the way a lot of other tests are written.